changelog.md

```diff
@@ -1,35 +1,17 @@
 # Changelog
 
-## 2.9.0~rc3
-
-### Journalist API
-* Added support for HTTP range request for submissions and replies endpoints (#7160, #7189)
-
-### Development
-* Ignored safety alerts 42926, 51385, 53048, 53868, 53869, 54219, 54421, 59473, 60026, 65505, 66667, 71591, 71594, 71595, 71608, 71680, 71681, 71684 (#7190)
-
-## 2.9.0~rc2
+## 2.9.0
 
 ### Web applications
+* Added accessibility improvements (#6536)
 * Dependency updates:
   * (Rust) sequoia-openpgp from 1.17.0 to 1.20.0 (#7178)
 
-### Operations
-* Added opt-in Tor PoW defense for the Source Interface (#7175)
-
-### Development
-* Ignored safety alert 70895 (#7179)
-* Added support for development virtualenv in Debian 12 (#7154)
-
-## 2.9.0~rc1
-
-### Web applications
-* Added accessibility improvements (#6536)
-
 ### Journalist API
-* Added support for HTTP range request for submissions and replies endpoints (#7160)
+* Added support for partial content requests for submissions and replies endpoints (#7160, #7189)
 
 ### Operations
+* Added opt-in Tor PoW defense for the Source Interface (#7175)
```

Comment:
Per <https://github.com/freedomofpress/securedrop-docs/blob/2c7d9ab8fd36626ec4edd5b91c4590f079dcd5a0/docs/admin/deployment/tor_pow.rst?plain=1#L13-L15>, I think this is worth making more explicit:

```suggestion
* Added Tor PoW defense for the Source Interface: opt-in for existing instances; on by default for new instances (#7175)
```

Or just `Added Tor PoW defense for the Source Interface`.

Reply:
Eh - that's a bit too much in the weeds for a changelog IMO. New instances are still explicitly required to make the choice in sdconfig so I'm comfortable with current language.

Reply:
That's fair.  I think you and I are parsing this language differently versus the current documentation—thus the second suggestion offering less rather than more detail to avoid that confusion altogether.  But if you're happy with this as it is I'll approve for your merge.